cms/app_registration.py

```diff
@@ -140,6 +140,14 @@ def configure_cms_apps(apps_with_features):
                 configure_app(app_config.cms_config)
 
 
+def ready_cms_apps(apps_with_features):
+    """Run ready() methods on every registered cms extension,
+    so that final checks can happen after all apps have been configured
+    """
+    for app_with_feature in apps_with_features:
+        app_with_feature.cms_extension.ready()
```

Comment:
@chaosk It would be nice to see all of the other apps configs at this point, this could help prevent apps having to use monkeypatching / hacking as much as they are by having exposure to their other apps and their own registered configs.